manager/controllers/utils.go

```diff
@@ -81,7 +82,6 @@ func ApplySort(c *gin.Context, tx *gorm.DB, fieldExprs database.AttrMap,
 
 func validateFilters(q QueryMap, allowedFields database.AttrMap) error {
 	for key := range q {
-		// system_profile is hadled by tags, so it can be skipped
```

Comment:
Why do you remove this comment?

Reply:
my bad, I must have somehow delete without noticing it, returning it